examples/demo_test.py

```diff
@@ -0,0 +1,29 @@
+import sys
+
+sys.path.insert(0, "./examples/env")
+sys.path.insert(0, "./zoo")
+import baseline
+import policies.keep_lane_agent
+
+import gym
+
+from smarts.zoo import registry
+from smarts.env.wrappers.episode_logger import EpisodeLogger
+from smarts.env.wrappers.record import RecordVideo
+from smarts.core.utils.episodes import episode_range
+
+# FormatObs should already be applied
+env = gym.make("figure_eight-v0")
+# gym.wrappers.Monitor
+env: gym.Env = RecordVideo(env, frequency=10, name="007-render")
```

Comment:
Will the `from smarts.env.wrappers.record import RecordVideo` solution work with vectorized environments?

Suggestion:
1. Currently our `env.render()` in `hiway_env.py` simply passes.
1. Move the image rendering to `env.render()`, whenever the camera-sensor is enabled in smarts.
1. Then, users can use smarts with the video recorders available from and widely used in gym based codes.
1. As a result, we will be more gym friendly and we do not need to build and maintain any of our own video recorder.
 
Examples of existing video recorders:
1. gym
    + video_recorder.py : https://github.com/openai/gym/blob/941e5bfea3abbea4a85a2fdc1752cdea28b36229/gym/wrappers/monitoring/video_recorder.py#L157

1. sb3 
    + Callback to display to tensorboard
    + Requires tensorboard, torch, moviepy
    + VideoRecorderCallback: https://stable-baselines3.readthedocs.io/en/master/guide/tensorboard.html#logging-videos 
 
1. sb3 
    + Video recorder environment wrapper
    + VecVideoRecorder: https://stable-baselines3.readthedocs.io/en/master/_modules/stable_baselines3/common/vec_env/vec_video_recorder.html#VecVideoRecorder

Reply:
Sure, I was trying to avoid keeping around state for `gym.render(..)` I could switch the `RecordVideo` wrapper to something like `CameraRender` to store the state needed pass out `render()` frames so that we can use `RecordVideo`.

Reply:
I was going about implementing this and found that the `gym` version that implements this properly is `gym>=0.18` due to this: https://github.com/openai/gym/pull/2139 ...

It conflicts with Colab because annoyingly they start Colab with `gym==0.17.3` so it requires a reload to use it.